client/asset/eth/eth.go

```diff
@@ -876,23 +890,25 @@ func (eth *ExchangeWallet) SignMessage(_ asset.Coin, msg dex.Bytes) (pubkeys, si
 // be the same as the hash of txData. contract is expected to be
 // (contractVersion|secretHash) where the secretHash uniquely keys the swap.
 func (eth *ExchangeWallet) AuditContract(coinID, contract, txData dex.Bytes, rebroadcast bool) (*asset.AuditInfo, error) {
-	tx := new(types.Transaction)
-	err := tx.UnmarshalBinary(txData)
-	if err != nil {
-		return nil, fmt.Errorf("AuditContract: failed to unmarshal transaction: %w", err)
-	}
+	/*
+		tx := new(types.DynamicTransaction)
+		err := tx.UnmarshalBinary(txData)
+		if err != nil {
+			return nil, fmt.Errorf("AuditContract: failed to unmarshal transaction: %w", err)
+		}
 
-	txHash := tx.Hash()
-	if !bytes.Equal(coinID, txHash[:]) {
-		return nil, fmt.Errorf("AuditContract: coin id != txHash - coin id: %x, txHash: %x", coinID, tx.Hash())
-	}
+		txHash := tx.Hash()
+		if !bytes.Equal(coinID, txHash[:]) {
+			return nil, fmt.Errorf("AuditContract: coin id != txHash - coin id: %x, txHash: %x", coinID, tx.Hash())
+		}
+	*/
```

Comment:
We are currently only sending the txdata as txData. This is not the whole transaction and shouldn't have enough info to recreate the tx. We could send the serialized tx instead, if it is necessary. It will be if we want to resend below.

Reply:
sending the serialized tx